### PR TITLE
manual fixes to u0000016

### PR DESCRIPTION
**Pull request by DanilSko**
Resolved some group characters, merged some duplicate character IDs, added some missing 'stage' and 'speaker' tags, fixed structure of the last act of the play

### CODE REVIEW

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml [hunk 1]
```diff
@@ -3101,314 +3102,316 @@
...
+            </sp>
+            <sp who="#ivan">
+              <speaker>Іван.</speaker>
+              <p>Я тепер з радощів аж затанцюю!<stage>(Приспівує й танцює)</stage>. Гандзю, Гандзю
+                кучерява, Де ти в біса ночувала?… Ой-ой-ой, кляте колінко, як запекло! Микита заридав і влав
+                на лаву.</p>
+            </sp>
+            <stage>Завіса</stage>
+          </div>
+          <div type='change'>
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml:3414]: Ok, then I'll make it the standard 'scene' and 'subscene' types here. In this play the first 4 acts (ДІЯ) have a standard division into scenes(Ява), and the fifth act (ДІЯ П'ЯТА) has an intermediary level: two 'changes' (Одміна перша, Одміна друга), inside which there are scenes (Ява)

[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: Done here: https://github.com/dracor-org/udracor/pull/5/commits/9129096f5f60e243824138e7a8eadbdf585b2726

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml [hunk 2]
```diff
@@ -1574,8 +1561,8 @@
             <speaker>Іван.</speaker>
             <p>Не гнівайтесь, ваше благородіє, зараз почнемо.</p>
           </sp>
-          <sp who="#spivajut">
-            <speaker>Співають.</speaker>
+          <sp who="#ivan #omelko #semen">
+            <stage>Співають.</stage>
             <p>%Та лугом іду, та коня веду. Розвивайся, луже. "Сватай мене, козаченьку, Коли любиш дуже!
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: Done here: https://github.com/dracor-org/udracor/pull/5/commits/71e192525cc980296f93a2b86552544bc21cbce2

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml [hunk 3]
```diff
@@ -848,33 +839,28 @@
...
               <l>Аж там грошей вже й чорт має ті вічі</l>
               <l>Нічим похмелитись!</l><stage>(двічі)</stage>
               <l>Ой піду я до шинкарки:</l>
               <l>"Всип, шинкарко, хоч з півкварти!"</l>
-              <l></l>
               <l>Шинкарочка к чорту дметься,</l>
               <l>Ще й з чумаченька сміється,</l>
-              <l>Дід що гіркий п'яниця!</l>
-              <stage>(двічі)</stage>
+              <l>Дід що гіркий п'яниця!</l><stage>(двічі)</stage>
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: Done here for all such cases: https://github.com/dracor-org/udracor/pull/5/commits/592876276a7e727e0f6290b12adf375e0f907157

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml [hunk 4]
```diff
@@ -905,8 +891,7 @@
           </sp>
           <sp who="#marusja">
             <speaker>Маруся.</speaker>
-            <p>Хіба ж я цього не знаю? Я замету між кочерги!<stage>(Зупинилась проти Микити і з жартом
-                говорить)</stage>. А ти, парубче, не сміти по хаті, а то я тебе ось цим деркачем по губах!
+            <p>Хіба ж я цього не знаю? Я замету між кочерги!<stage>(Зупинилась проти Микити і з жартом говорить)</stage>. А ти, парубче, не сміти по хаті, а то я тебе ось цим деркачем по губах!
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: Done here for all such cases: https://github.com/dracor-org/udracor/pull/5/commits/592876276a7e727e0f6290b12adf375e0f907157

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml [hunk 5]
```diff
@@ -564,7 +555,7 @@
             <stage>(підійшов до парубоцтва)</stage>
             <p>Добривечір вам, панове товариство!</p>
           </sp>
-          <sp who="#usi">
+          <sp who="#omelko #1-j_parubok #2-j_parubok #3-j_parubok #4-j_parubok #5-j_parubok #6-j_parubok #7-j_parubok">
```
[the remark this answers is not in the document]
Comment (DanilSko) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: oh, true, my bad, thank you! fixed now, see here: https://github.com/dracor-org/udracor/pull/5/commits/1eb56983d0ac80e6a766423c2a5e1970c4ef588b

**Comment by DanilSko** [comment]
Ok, @cmil I think this list of mistakes is corrected. Many thanks for your review! Is there anything else I could fix?

File: tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmil) [reply tei/kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml]: The DraCor API will probably only pick up on the scenes then. Could you open an issue in [dracor-api](https://github.com/dracor-org/dracor-api/issues) if we need to consider the sub scenes as separate segments.

**Comment by cmil** [comment]
@DanilSko Thanks for the changes! The only technical issue I can see with `kropivnitskij-daj-sertsju-volju-zavede-u-nevolju.xml` now is the missing `publicationStmt`, which prevents it from passing TEI all validation. But this seems to be true for all the other documents as well and could be fixed in a separate step.

**Pull request merged**